object-manipulation-2/src/pick.js

```diff
@@ -0,0 +1,16 @@
+/* exported pick */
+
+function pick(source, keys) {
+  var newObj = {};
+
+  for (var i = 0; i < keys.length; i++) {
+    for (var key in source) {
```

Comment:
Same thing here, no need for a `for...in` loop, just use the current value in the loop as the property name to check.